articles/blocks-and-scopes.html

```diff
@@ -26,7 +26,7 @@ <h3>Code Blocks</h3>
 <li>each file has a <b>file block</b> containing all the source code,
 	including the package import declarations, in that file.</li>
 <li>generally, a pair of braces <code>{}</code> encloses a <b>local block</b>.
-	However, some local blocks don't enclosed with <code>{}</code>, such blocks are called implicit local blocks.
+	However, some local blocks don't enclose with <code>{}</code>, such blocks are called implicit local blocks.

```

Comment:
Here, is "are enclosed within" better?

Reply:
Because I don't know any Go I wasn't sure what you meant. Thinking of it "are enclosed within" makes sense. I'll fix it.